pyqtorch/modules/parametric.py

```diff
@@ -38,28 +34,10 @@ def forward(self, state: torch.Tensor, thetas: torch.Tensor) -> torch.Tensor:
         return self.apply(mats, state)
 
     def extra_repr(self) -> str:
-        return f"qubits={self.qubits}, n_qubits={self.n_qubits}"
+        return super().extra_repr()
```

Comment:
All of those methods can be removed completely now, because you are only calling the inherited function from `AbstractGate` anyway:)

Reply:
Right, I initially made it an abstract method until I realized they all had the same method. I forgot to remove all of those afterward.